dwh/indicators.py

```diff
@@ -122,6 +122,7 @@ def find_date_range(args: argparse.ArgumentParser) -> Tuple[str, str]:
   patient_agg_obs_df.cache()
   VL_df_P = indicator_lib.calc_TX_PVLS(
       patient_agg_obs_df, VL_code='5090AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      failure_threshold=30,
```

Comment:
With this updated value, you need to update the test too [here](https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/blob/da4716550e10d71b485576ed8214860364ebaa80/dwh/validate_indicators.sh#L58). It is currently [failing](https://travis-ci.org/github/GoogleCloudPlatform/openmrs-fhir-analytics/jobs/774627694#L7634).

Reply:
yes, Apologies, this was not considered. Changing it appropriately